2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,12 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+const myFavoriteRecipe = {
+  title: 'Fajita wraps',
+  servings: 8,
+  ingredients: ['tortillas', 'chicken', 'mixed peppers', 'lettuce', 'cheese', 'potatoes']
+};
+
+console.log(myFavoriteRecipe.title);
+console.log(myFavoriteRecipe.servings);
+console.log(myFavoriteRecipe.ingredients);
```

Comment:
it needs five recipes and I think you should use a function to log it out.

Reply:
I agree, you can use an array method here to log the ingredients one by one, e.g. `forEach`